streams/src/commonMain/kotlin/com/mirego/trikot/streams/reactive/Publishers.kt

```diff
@@ -28,6 +28,11 @@ object Publishers {
     fun <T> completed(): Publisher<T> {
         return PublishSubjectImpl<T>().also { it.complete() }
     }
+
+    @JsName("error")
+    fun <T> error(error: Throwable): Publisher<T> {
```

Comment:
Wow! I added the same method in https://github.com/mirego/trikot.streams/pull/56 😄

Reply:
ah! Sorry for the merge conflict then :P